src/include/sof/audio/component.h

```diff
@@ -152,6 +152,12 @@ enum comp_endpoint_type {
 	COMP_ENDPOINT_NODE
 };
 
+ /* \brief Type of component copy, which can be changed on runtime */
+enum comp_copy_type {
+	COMP_COPY_NORMAL = 0,
+	COMP_COPY_BLOCKING
```

Comment:
Comma after COMP_COPY_BLOCKING because in the future we might have other copy types.